addons/html_editor/static/src/main/media/media_dialog/document_selector.xml

```diff
@@ -1,10 +1,10 @@
 <templates id="template" xml:space="preserve">
 <t t-name="html_editor.DocumentAttachment">
-    <div class="o_existing_attachment_cell o_we_attachment_highlight card col-2 position-relative mb-2 p-2 opacity-trigger-hover cursor-pointer" t-att-class="{ o_we_attachment_selected: props.selected }" t-on-click="props.selectAttachment">
+    <button class="o_existing_attachment_cell o_we_attachment_highlight card col-2 position-relative mb-2 p-2 opacity-trigger-hover cursor-pointer" t-att-class="{ o_we_attachment_selected: props.selected }" t-on-click="props.selectAttachment">
         <RemoveButton remove="() => this.remove()"/>
         <div t-att-data-url="props.url" role="img" t-att-aria-label="props.name" t-att-title="props.name" t-att-data-mimetype="props.mimetype" class="o_image d-flex align-items-center justify-content-center"/>
         <small class="o_file_name d-block text-truncate" t-esc="props.name"/>
-    </div>
+    </button>
```

Comment:
Not convinced by having a button in a button. I think this needs deeper changes to make it good.